### PR TITLE
add function in populate_db to create an event.

### PR DESCRIPTION
**Pull request by Anupam-dagar**
This adds a function to populate the db with an event.
@CosmicCoder96 I have included only the important fields to be filled up. Should I include any other field? Also this created event shows up in db but not on the frontend. I didn't find the user in the init function of the model. How can I associate it with a user (probably admin user)
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by Anupam-dagar** [comment]
@CosmicCoder96 can you confirm if the failing tests are due to my changes, didn't seems like it.

File: populate_db.py [hunk 1]
```diff
@@ -316,13 +323,32 @@ def create_admin_message_settings():
 
 def create_custom_placeholders():
     custom_placeholder, _ = get_or_create(
-        CustomPlaceholder, name='Hills',
+        CustomPlaceholder, name='abc',
```
Comment (CosmicCoder96) [inline populate_db.py]: Why? 
and yes, it looks like this PR is failing the build.

Comment (Anupam-dagar) [reply populate_db.py]: My bad, I left it in place after doing the test.
I'll fix the tests.

**Comment by Anupam-dagar** [comment]
@CosmicCoder96 I rebased my development branch and ran the tests locally using `python3 -m unittest discover tests/all/`
4 tests are failing locally.

**Comment by Anupam-dagar** [comment]
I ran the tests again by using testdb this time. Now there are 2 failing tests on development branch
Following is the traceback:
```shell
FAIL: test_get_or_create (integration.api.helpers.test_db.TestDBHelperValidation)
Method to test the function get_or_create
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/anupam/github/open-event-server/tests/all/integration/api/helpers/test_db.py", line 51, in test_get_or_create
    self.assertEqual(event.id, obj.id)
AssertionError: 7 != 1

======================================================================
FAIL: test_safe_query_exception (integration.api.helpers.test_db.TestDBHelperValidation)
Method to test the exception in function safe_query
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/anupam/github/open-event-server/tests/all/integration/api/helpers/test_db.py", line 42, in test_safe_query_exception
    self.assertRaises(ObjectNotFound, lambda: safe_query(db, Event, 'id', 1, 'event_id'))
AssertionError: ObjectNotFound not raised by <lambda>
```

**Comment by Anupam-dagar** [comment]
@CosmicCoder96 I have setup the server again. At first I ran the tests on the `development` branch using `python3 -m unittest discover tests/all/`, this failed 2 tests, following is the traceback:
```shell
======================================================================
FAIL: test_get_or_create (integration.api.helpers.test_db.TestDBHelperValidation)
Method to test the function get_or_create
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/anupam/github/open-event-server/tests/all/integration/api/helpers/test_db.py", line 51, in test_get_or_create
    self.assertEqual(event.id, obj.id)
AssertionError: 105 != 1

======================================================================
FAIL: test_safe_query_exception (integration.api.helpers.test_db.TestDBHelperValidation)
Method to test the exception in function safe_query
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/anupam/github/open-event-server/tests/all/integration/api/helpers/test_db.py", line 42, in test_safe_query_exception
    self.assertRaises(ObjectNotFound, lambda: safe_query(db, Event, 'id', 1, 'event_id'))
AssertionError: ObjectNotFound not raised by <lambda>
```

Then I shifted to my `eventpopulate` branch and it failed two more tests as follows:
```shell
FAIL: test_get_count (integration.api.helpers.test_db.TestDBHelperValidation)
Method to test the number of queries concerning a Model
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/anupam/github/open-event-server/tests/all/integration/api/helpers/test_db.py", line 64, in test_get_count
    self.assertEqual(get_count(TicketHolder.query), 1)
AssertionError: 3 != 1

FAIL: test_should_create_attendee_forms (integration.api.helpers.test_events.TestEventUtilities)
Method to test custom forms for attendees of an event.
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/anupam/github/open-event-server/tests/all/integration/api/helpers/test_events.py", line 22, in test_should_create_attendee_forms
    self.assertEqual(get_count(CustomForms.query), 3)
AssertionError: 9 != 3
```

Now 4 tests fail on the development branch too.
Also as written in the readme, this is the command to run the tests `python3 -m unittest discover tests/all/
`, right?

I did set the `APP_CONFIG` to `testingconfig`.

**Comment by uds5501** [comment]
@Anupam-dagar Regarding your first two errors, I would like to point out that currently the tests have been configured in such a way that they should correspond to the fact that an event does not exist initially. But since you are creating one in the initialization stage itself, it breaks the tests.

I think you should consult @iamareebjamal and @srv-twry , the contributors and change the test behaviors accordingly

**Comment by Anupam-dagar** [comment]
@uds5501 the first two errors are on the development branch where I haven't added the db script. Also I recreated the db.

**Comment by iamareebjamal** [comment]
Errors can't be on development branch since the build is passing

**Comment by Anupam-dagar** [comment]
@iamareebjamal I recloned the repo and recreated the db. Still it was failing. Can you tell me the steps you follow to run the tests.

**Comment by iamareebjamal** [comment]
https://github.com/fossasia/open-event-server/blob/a04dbc3992c9cbc07bff918f5de9ebba76edbaa9/.travis.yml#L31-L38

**Comment by prateekj117** [comment]
@Anupam-dagar Still on this? If not close it, I will make a new PR regarding the issue.

**Comment by mariobehling** [comment]
Thank you for engaging in the project. Unfortunately, you have not followed up on the comments and questions raised. It would be great, if you follow up at a later time and open a new PR. Closing this now due to inactivity.

**Comment by prateekj117** [comment]
@mariobehling Is this feature needed. if yes, I would open an issue regarding this and would work on it.

**Comment by Anupam-dagar** [comment]
@prateekj117 This was more of a development help to create test events faster and not a user side feature.

**Comment by prateekj117** [comment]
@Anupam-dagar Yaa, I get that. Still, the same question, do we need it? If yes, let's open a corresponding issue and I would like to work on it.

**Comment by Anupam-dagar** [comment]
It's good if we can have it.

**Pull request closed without merging**